app/models/course/assessment.rb

```diff
@@ -86,6 +86,26 @@ class Course::Assessment < ApplicationRecord
     end
   end)
 
+  # Used by the with_actable_types scope in Course::LessonPlan::Item.
+  # Edit this to remove items for display.
+  #
+  # Settings for which assessment tabs to show are stored in course settings with the nested keys:
+  # (:course_assessments_component, :lesson_plan_items, :tab_<tab_id>).enabled
+  scope :for_display, (lambda do |settings|
+    disabled_tab_keys = []
+    lesson_plan_item_keys = settings.settings(Course::AssessmentsComponent.key).lesson_plan_items
+
+    if lesson_plan_item_keys
+      disabled_tab_keys = lesson_plan_item_keys.keys.reject do |tab|
```

Comment:
This implicitly assumes `enabled_by_default` is true. I don't foresee `enabled_by_default` will be very useful for lesson plan. There is also 'visible_by_default` which we have not yet implemented (don't think it is useful either) which we should differentiate it form.

If we keep `enabled_by_default`, then we should check its value here.